dashboard/pages/Steam.py

```diff
@@ -1,11 +1,74 @@
 '''A script that creates the Steam page for the dashboard.'''
 
-from os import environ as ENV
+from re import match
+from os import environ as ENV, listdir
+from datetime import datetime, timedelta
 
+import pandas as pd
+import altair as alt
 import streamlit as st
+from boto3 import client
+from dotenv import load_dotenv
+
+from psycopg2 import connect
+from psycopg2.extras import RealDictCursor
+from psycopg2.extensions import connection
+
+
+def get_db_connection(config) -> connection:
+    """Returns a connection to the database."""
+
+    return connect(
+        dbname=config["DB_NAME"],
+        user=config["DB_USER"],
+        password=config["DB_PASSWORD"],
+        host=config["DB_HOST"],
+        port=config["DB_PORT"],
+        cursor_factory=RealDictCursor
+    )
+
+
+def metric_games_yest(conn: connection) -> pd.DataFrame:
```

Comment:
Small detail; it should be more explicit that this is collecting yesterday's game data. Preferrably mention that in the docstring?